### DISCUSSION TITLE
Disable auto-repeat

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#3093] [Features Request & Ideas | 特性请求 & 头脑风暴]
Can we add the feature of playing just once the playlist?

Every time I share a playlist with others (and relax into it) I'm really stressed because I need to MANUALLY stop the player when the playlist finishes. I feel it's a very basic feature, but I'm willing to consider it if the development team doesn't feel it's a good idea.

I offer my self as a C++ developer to develop this feature if you like.

**Comment by BLumia** [comment]
Sorry for the late reply since we were not checking issues that were created outside the developer-center repo.

😂 the user who report this issue already deleted their account, but it's still a valid feature request. Anyway, I'll mark this as a feature request and see if we can introduce this feature in the main product, and feel free to fork the project and start hacking if you like :)

**Comment by meiyixiang** [comment]
Your feedback has been recorded and internally assessed
